File: level1/약수의_합/Solution.java
class Solution {
    public int solution(int n) {
        int answer = 0;
        for (int i = 1; i <= n / 2; i++) {
            if (n % i == 0)
                answer += i;
        }
        return answer + n;
    }

    public static void main(String[] args) {
        Solution s = new Solution();
        System.out.println(s.solution(12));
        System.out.println(s.solution(5));
    }
}

File: level1/약수의_합/solution.js
function solution(n) {
  var answer = 0;
  for (var i = 1; i <= n / 2; i++) {
    if (n % i === 0) answer += i;
  }
  return answer + n;
}

console.log(solution(12));
console.log(solution(5));
